code/datums/keybindings/emote.dm

```diff
@@ -592,3 +592,46 @@
 /datum/keybinding/emote/simple_animal/pet/cat/purr
 	linked_emote = /datum/emote/living/simple_animal/pet/cat/purr
 	name = "Purr (Cat)"
+
+/datum/keybinding/custom
+	category = KB_CATEGORY_EMOTE_CUSTOM
+	var/emote_text = "Insert custom me emote text."
+
+/datum/keybinding/custom/down(client/C)
+	. = ..()
+	if(emote_text == initial(emote_text))
+		return
+	C.mob.me_verb(emote_text)
+
+/datum/keybinding/custom/can_use(client/C, mob/M)
+	return isliving(M) && ..()
+
+/datum/keybinding/custom/one
+	name = "Custom Emote 1"
+
+/datum/keybinding/custom/two
+	name = "Custom Emote 2"
+
+/datum/keybinding/custom/three
+	name = "Custom Emote 3"
+
+/datum/keybinding/custom/four
+	name = "Custom Emote 4"
+
+/datum/keybinding/custom/five
+	name = "Custom Emote 5"
+
+/datum/keybinding/custom/six
+	name = "Custom Emote 6"
+
+/datum/keybinding/custom/seven
+	name = "Custom Emote 7"
+
+/datum/keybinding/custom/eight
+	name = "Custom Emote 8"
+
+/datum/keybinding/custom/nine
+	name = "Custom Emote 9"
+
+/datum/keybinding/custom/ten
+	name = "Custom Emote 10"
```

Comment:
No way in hell are we giving people 10 of these.

3 for regulars, may more for patrons

Reply:
This needs wider discussion I think. Once a solid figure is presented I am happy to implement.

Reply:
Maybe this could be a good bonus for byond members as well?

Reply:
done